app/javascript/packs/selectors/answersSelector.js

```diff
@@ -1,3 +1,19 @@
 export function getAnswerForQuestionAndBuilding(questionId, buildingId, state) {
   return state.buildings[buildingId].answers[questionId]
 }
+
+//##returns (from the questions provided) how many are remaining to answer
+//##if no buildingId is provided, we are not viewing a specific building
+// so we should not render how many questions are remaining for a building
+//## if no questions are provided, the building did not have any questions for the users so they have no questions
+//to answer for that building
+export function getRemainingAnswersforCategory(questions, buildingId, state) {
+  return questions.reduce((count, question) => {
+    let answer = state.buildings[buildingId].answers[question.id];
+    if (!answer.text.trim() && !answer.attachment_file_name) {
```

Comment:
Be sure to check if `answer` is undefined here. It's possible that we don't yet have an answer (in the case of an Asset Manager, delegating questions for the first time).